timemachine/cpp/src/kernels/k_nonbonded_pairs.cuh

```diff
@@ -0,0 +1,202 @@
+#pragma once
```

Comment:
We may want to add upfront that the order of associative operations in this file must be consistent with that of the k_nonbonded.cuh file in order for the exclusions to cancel out correctly.

Reply:
good call; done in 4a9d33d